doc/source/master_list.bib

```diff
@@ -847,6 +847,28 @@ @Article{Roberts15
   pages = {211-228},
   url = {http://dx.doi.org/10.3189/2015AoG69A760}
 }
+
+@Article{Kimmritz15
+  author = "M. Kimmritz and S. Danilov and M. Losch",
+  title = "{On the convergence of the modified elastic-viscous-plastic method for solving the sea ice momentum equation}",
+  journal = JCP,
+  year = {2015},
+  volume = {296},
+  pages = {90-100},
+  url = {http://dx.doi.org/10.1016/j.jcp.2015.04.051}
+}
+
+@Article{Lemieux12
```

Comment:
Can you make sure to add these both sequentially (e.g. 2012 before 2015) and alphabetically?

Reply:
```I think that the alpa,beta < 1 should have been alpha, beta > 1 in  the documentation part of the revised evp